### ISSUE TITLE
The rich text editor cannot be opened in wechat's built-in browser

### ISSUE THREAD

**Issue by hwjfqr** [tinymce/tinymce#9226]
## 📝 Provide detailed reproduction steps (if any)
I opened https://www.tiny.cloud/docs/tinymce/6/ using the built-in browser of the WeChat mobile app, and found that the rich text editor did not display properly. In China, everyone uses WeChat, so it is crucial to adapt to the WeChat built-in browser.
Your prompt attention to this matter would be greatly appreciated.

### ✔️ Expected result
The rich text editor should display correctly.
![e67f23ef82f43d7e94733b7385aabed](https://github.com/tinymce/tinymce/assets/22411284/d8aa9dc1-7f95-41b9-8733-df9e1bbe67c4)

### ❌ Actual result
The rich text editor is not displayed.
![dea1ed9e690fa8fcd39d46c9792c1da](https://github.com/tinymce/tinymce/assets/22411284/b07a1f4e-154d-4ddb-91d6-7c176f75d628)

## 📃 Other details
- Browser：WeChat built-in browser on mobile (WeChat version 8.0.44)
- OS：iOS 17、iOS16
- First affected version：tinymce 6.7.3
- Worked in version：tinymce 6.7.3

**Comment by lostkeys** [comment]
Hi @hwjfqr, can you (or other wechat users seeing this) manage to provide console output or debug this issue? Can it be replicated on a vanilla TinyMCE instance (eg. to make sure it is not a issue with out website demos)

**Comment by hwjfqr** [comment]
> Hi @hwjfqr, can you (or other wechat users seeing this) manage to provide console output or debug this issue? Can it be replicated on a vanilla TinyMCE instance (eg. to make sure it is not a issue with out website demos)

Hello @lostkeys  , I can confirm that this is not just an issue with the demo website.  I use the TinyMCE editor in a CMS system that I'm developing, and I also encounter the issue of it not displaying in the WeChat embedded browser.  Based on my testing on real devices, there are no error messages in the console.

It's worth noting that, according to my current testing, this issue only occurs on iOS systems (I tested on iOS 16 and iOS 17).

Thank you for your attention.

![cd1051f6f379b110af2f395e5cd0b73](https://github.com/tinymce/tinymce/assets/22411284/659044e1-e8a4-498d-b78f-13cfa44d53c3)
![ed5e205cedcc68501ef57e1101718fc](https://github.com/tinymce/tinymce/assets/22411284/7b8f3534-f8ef-4fb3-959b-cdcb782afbbf)

**Comment by TheSpyder** [comment]
Errors may not show up in an on-device console, it probably needs to be in Safari developer tools (on mac, with the iOS device connected via USB).

**Comment by yeweilulu** [comment]
I experienced the same problem when using tinymce 6.8.2. Is this issue fixed

**Comment by YeZhang24** [comment]
The issue still persists with the TinyMCE 7.0.0 version I'm using. Is there a solution available?

**Comment by smalljop** [comment]
The issue still persists with the TinyMCE 7.0.0 version I'm using. Is there a solution available?

**Comment by TheSpyder** [comment]
Sorry, it is difficult for us to replicate this issue. We need help getting a browser console log when it happens - in theory it should be possible to plug your iphone into any mac and use Safari's developer tools to access this.

**Comment by smalljop** [comment]
> Sorry, it is difficult for us to replicate this issue. We need help getting a browser console log when it happens - in theory it should be possible to plug your iphone into any mac and use Safari's developer tools to access this.

Hello, I have observed that tinymce5 can be used normally, but tinymce6 or 7 are abnormal. Is there any change in version 6?

**Comment by TheSpyder** [comment]
yes there are a lot of changes, unfortunately too many to suggest which one might be causing this problem.

**Comment by smalljop** [comment]
> yes there are a lot of changes, unfortunately too many to suggest which one might be causing this problem.

I guess it has something to do with the browser kernel? This is the environment I can reproduce

<img width="329" alt="1725070520426" src="https://github.com/user-attachments/assets/bcdff0c1-81ef-4a97-98c2-7177f46507fc">

**Comment by ArvinJ-H** [comment]
Hi @hwjfqr, we can reliably replicate the issue but haven’t been able to capture error logs. Using official WeChat developer tools, the site renders correctly, making it difficult to pinpoint the cause.

Given that TinyMCE 5 works while 6 and 7 do not, this may be related to changes introduced in v6 or the WeChat browser kernel. We’re considering filing a report with WeChat, and if you have additional insights or error logs, they’d be helpful in debugging.

Let us know if there’s anything else we can try.

<img width="1251" alt="Image" src="https://github.com/user-attachments/assets/86c9d5c2-9493-4675-bb8c-30e8bb7312d9" />

**Comment by superliebe** [comment]
The issue still persists with the TinyMCE 7.7.1 version I'm using. Is there a solution available?
